### PR TITLE
Fix issues with variables initializations and nullptr handling; Resolve build warnings

### PR DESCRIPTION
**Pull request by kotreshr**

### CODE REVIEW

File: openvino_tensorflow/kernels/encapsulate_op.cc [hunk 1]
```diff
@@ -718,7 +719,7 @@ Status NGraphEncapsulateOp::GetExecutable(
 
 Status NGraphEncapsulateOp::Fallback(OpKernelContext* ctx) {
   OVTF_VLOG(1) << "Cluster " << name() << " fallback to native TF runtime ";
-  int64_t start_ns;
+  int64_t start_ns = 0, duration_in_ms;
```
Comment (sspintel) [inline openvino_tensorflow/kernels/encapsulate_op.cc:722]: Shouldn't we be initializing duration_in_ms to 0 as well?

**Review by sspintel — APPROVED** [review]
LGTM, can merge once build/sanity check is done.

File: openvino_tensorflow/kernels/encapsulate_op.cc (hunk 1 above)
Comment (kotreshr) [reply openvino_tensorflow/kernels/encapsulate_op.cc]: Yes, added.

**Review by sspintel — APPROVED** [review]
LGTM

**Pull request merged**